src/atomicdex/app.cpp

```diff
@@ -330,13 +331,22 @@ namespace atomic_dex
     application::application(QObject* pParent) noexcept :
         QObject(pParent),
         m_update_status(QJsonObject{
-            {"update_needed", false}, {"changelog", ""}, {"current_version", ""}, {"download_url", ""}, {"new_version", ""}, {"rpc_code", 0}, {"status", ""}}),
-        m_manager_models{
-            {"addressbook", new addressbook_model(system_manager_.create_system<qt_wallet_manager>(), this)},
-            {"orders", new orders_model(this->system_manager_, this->dispatcher_, this)},
-            {"internet_service", std::addressof(system_manager_.create_system<internet_service_checker>(this))},
-            {"notifications", new notification_manager(this->dispatcher_, this)}}
+            {"update_needed", false}, {"changelog", ""}, {"current_version", ""}, {"download_url", ""}, {"new_version", ""}, {"rpc_code", 0}, {"status", ""}})
     {
+        //! Creates managers
+        {
+            system_manager_.create_system<qt_wallet_manager>();
+            system_manager_.create_system<addressbook_manager>(system_manager_);
+        }
+        
+        //! Creates models
+        {
+            m_manager_models.emplace("addressbook", new addressbook_model(system_manager_, this));
```

Comment:
This is a good approach, however later we will modified this to became all systems for every models, addressbook page will appears and will contains `addressbook` model in order to respect existing architecture